components/FourYearPlanPage.tsx

```diff
@@ -680,7 +680,6 @@ export const FourYearPlanPage: FC<FourYearPlanType> = memo(
         undo = false;
       }
     }
-
```

Comment:
Can we get this line back? It's quite nice for separation between the function body and the JSX return value.